db/migrate/20231201171351_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.references :author, foreign_key: { to_table: :users }
+      t.integer :comments_counter, default: 0
+
```

Comment:
- [ ] Please follow the requirements properly, no doubt you implemented almost everything but you forgot to add the `likes_counter` for the post. Kindly take a look at the ERD diagram, even I added the screenshot below to look carefully at what you missed and fulfil the requirements.

![image](https://github.com/EvansSnave/Blog_app_rails/assets/66476329/4c4fd7cf-887c-478e-bf07-231f9769b4e1)


### Requirement
![image](https://github.com/EvansSnave/Blog_app_rails/assets/66476329/817d5b54-1b55-4ad8-b928-814bbda4ed77)